www-client/seamonkey/seamonkey-2.53.11.ebuild

```diff
@@ -0,0 +1,585 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+WANT_AUTOCONF="2.1"
+
+PYTHON_COMPAT=( python3_{8..10} )
+PYTHON_REQ_USE='ncurses,sqlite,ssl,threads(+)'
+
+# This list can be updated with scripts/get_langs.sh from the mozilla overlay
+# note - could not roll langpacks for: ca fi
+#MOZ_LANGS=(ca cs de en-GB es-AR es-ES fi fr gl hu it ja lt nb-NO nl pl pt-PT
+#	    ru sk sv-SE tr uk zh-CN zh-TW)
+MOZ_LANGS=(cs de en-GB es-AR es-ES fr hu it ja lt nl pl pt-PT
+	    ru sk sv-SE zh-CN zh-TW)
+
+MOZ_PV="${PV/_pre*}"
+MOZ_PV="${MOZ_PV/_alpha/a}"
+MOZ_PV="${MOZ_PV/_beta/b}"
+MOZ_PV="${MOZ_PV/_rc/rc}"
+MOZ_P="${P}"
+MY_MOZ_P="${PN}-${MOZ_PV}"
+PATCH="${PN}-2.53.11-patches-01"
+
+if [[ ${PV} == *_pre* ]] ; then
+	MOZ_HTTP_URI="https://archive.mozilla.org/pub/${PN}/candidates/${MOZ_PV}-candidates/build${PV##*_pre}"
+else
+	MOZ_HTTP_URI="https://archive.mozilla.org/pub/${PN}/releases/${MOZ_PV}"
+fi
+
+SRC_URI="${MOZ_HTTP_URI}/source/${MY_MOZ_P}.source.tar.xz -> ${P}.source.tar.xz
+	${MOZ_HTTP_URI}/source/${MY_MOZ_P}.source-l10n.tar.xz -> ${P}.source-l10n.tar.xz
+	https://github.com/BioMike/gentoo-${PN}-patches/archive/refs/tags/${PV}.tar.gz -> ${PATCH}.tar.gz"
+
+S="${WORKDIR}/${MY_MOZ_P}"
+
+MOZ_GENERATE_LANGPACKS=1
+MOZ_L10N_SOURCEDIR="${S}/${P}-l10n"
+inherit autotools check-reqs desktop edos2unix flag-o-matic mozcoreconf-v6 mozlinguas-v2 pax-utils toolchain-funcs xdg-utils
+
+DESCRIPTION="Seamonkey Web Browser"
+HOMEPAGE="https://www.seamonkey-project.org/"
+
+LICENSE="MPL-2.0 GPL-2 LGPL-2.1"
+SLOT="0"
+SYSTEM_IUSE=( +system-{av1,harfbuzz,icu,jpeg,libevent,libvpx,png,sqlite} )
+IUSE="+chatzilla cpu_flags_arm_neon +crypt dbus debug +gmp-autoupdate +ipc jack
+lto pulseaudio selinux startup-notification test wifi"
+IUSE+=" ${SYSTEM_IUSE[@]}"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+
+RESTRICT="!test? ( test )"
+
+ASM_DEPEND=">=dev-lang/yasm-1.1"
```

Comment:
This seems redundant. Did this maybe sometime plan to support nasm too?

Reply:
I agree, this could have some improvement. This is a part that I haven't touched yet after taking over the ebuild from the previous maintainer.

Reply:
Removed redundancy in latest commit.